Backend/CourseService.cs

```diff
@@ -11,6 +13,7 @@ namespace FRITeam.Swapify.Backend
     public class CourseService : ICourseService
     {
         private readonly IMongoDatabase _database;
+        private IMongoCollection<Course> _courseCollection => _database.GetCollection<Course>(nameof(Course));
```

Comment:
Also properties we write with _loweCamelCase?

Reply:
I use _ for private field and private propetries, but we can discute about convention on wednesday

Reply:
So you can merge it for now... Refactor can be later